### PR TITLE
Timer styling

### PR DESCRIPTION
**Pull request by timodilg**

### CODE REVIEW

**Comment by arisslin** [comment]
After npm start the app opens and i get this error message, but the app seems to work:
Bildschirmfoto 2020-03-17 um 18 06 45

For test i deleted in the index.html the link to the missing file, but the error occurs again.

File: src/components/TimerFocusTask.js [hunk 1]
```diff
@@ -4,21 +4,27 @@ import styled from 'styled-components/macro'
...
-      </span>
+      <input type="checkbox" />
+      <span>Hemden bügeln</span>
     </TimerFocusTaskStyled>
   )
 }
 
 const TimerFocusTaskStyled = styled.li`
-  border: 1px solid rgb(207, 207, 207);
+  /* border: 1px solid rgb(207, 207, 207); */
```
Comment (arisslin) [inline src/components/TimerFocusTask.js]: Delte comments for code you not need.

File: src/components/TimerIcons.js [hunk 2]
```diff
@@ -3,14 +3,18 @@ import { Link } from 'react-router-dom'
...
+  const todoIcon = require('../images/todolist-white-75.svg')
 
   return (
     <TimerIconsStyled>
+      <Link to="/todo">
+        <img src={todoIcon} alt="todolist" width="30" />
+      </Link>
       <img src={soundOnIcon} alt="sound on" width="30" />
-      <img src={soundOffIcon} alt="sound off" width="30" />
+      {/* <img src={soundOffIcon} alt="sound off" width="30" /> */}
```
Comment (arisslin) [inline src/components/TimerIcons.js]: Delete comments for code you not need.

File: src/components/TimerIcons.js [hunk 3]
```diff
@@ -3,14 +3,18 @@ import { Link } from 'react-router-dom'
 import styled from 'styled-components/macro'
 
 export default function TimerIcons() {
-  const soundOnIcon = require('../images/sound-on.svg')
-  const soundOffIcon = require('../images/sound-off.svg')
-  const settingsIcon = require('../images/settings.svg')
+  const soundOnIcon = require('../images/sound-on-white.svg')
+  // const soundOffIcon = require('../images/sound-off-white.svg')
```
Comment (arisslin) [inline src/components/TimerIcons.js]: Delete comments for code you not need.

File: src/components/TimerIcons.js [hunk 4]
```diff
@@ -3,14 +3,18 @@ import { Link } from 'react-router-dom'
...
+  const settingsIcon = require('../images/settings-white.svg')
+  const todoIcon = require('../images/todolist-white-75.svg')
 
   return (
     <TimerIconsStyled>
+      <Link to="/todo">
+        <img src={todoIcon} alt="todolist" width="30" />
```
Comment (arisslin) [inline src/components/TimerIcons.js]: The alt attribute is for the user. When the image is not loaded he can read a text which describes what he sees. I think "Icon for todo list" is better.

Maybee you can set the width by css with styled components. Than you can reuse the settings for several icons.

File: src/pages/Timer.js [hunk 5]
```diff
@@ -6,6 +6,9 @@ import TimerDisplay from '../components/TimerDisplay'
 // import Sound from '../components/Sound'
```
Comment (arisslin) [inline src/pages/Timer.js]: Delete comments for code you not need.

File: src/pages/Timer.js [hunk 6]
```diff
@@ -6,6 +6,9 @@ import TimerDisplay from '../components/TimerDisplay'
...
 import TimerIcons from '../components/TimerIcons'
 import TimerFocusTask from '../components/TimerFocusTask'
+import play from '../images/play.svg'
+// import pause from '../images/pause.svg'
```
Comment (arisslin) [inline src/pages/Timer.js]: Delete comments for code you not need.

File: src/pages/Timer.js [hunk 7]
```diff
@@ -82,35 +81,59 @@ class Timer extends Component {
...
+          <section>
+            <TimerDisplay currentTime={this.state.currentTime} />
+          </section>
+
+          <div>
+            {this.state.timer ? null : (
+              <img src={play} alt="play button" onClick={this.startTimer} />
+            )}
+
+            {/* <img src={pause} height="50px" alt="pause button" /> */}
```
Comment (arisslin) [inline src/pages/Timer.js]: Delete comments for code you not need.

File: src/pages/Timer.js [hunk 8]
```diff
@@ -6,6 +6,9 @@ import TimerDisplay from '../components/TimerDisplay'
 // import Sound from '../components/Sound'
 import TimerIcons from '../components/TimerIcons'
 import TimerFocusTask from '../components/TimerFocusTask'
+import play from '../images/play.svg'
+// import pause from '../images/pause.svg'
+import stop from '../images/stop.svg'
 
 class Timer extends Component {
```
Comment (arisslin) [inline src/pages/Timer.js]: Think about functional components. I don't know if it makes sense to mix them with class components in a project. Maybee Jerry oder Jan has an answer. Let me know i am interestet in this.

File: src/pages/Timer.js [hunk 9]
```diff
@@ -82,35 +81,59 @@ class Timer extends Component {
...
+            )}
+
+            {/* <img src={pause} height="50px" alt="pause button" /> */}
+
+            {this.state.timer ? (
+              <img src={stop} alt="stop button" onClick={this.stopTimer} />
+            ) : null}
+          </div>
+          <TimerIcons />
+          {/* <Sound soundOn={this.state.timerState === timerStates.RUNNING} /> */}
```
Comment (arisslin) [inline src/pages/Timer.js]: Delete comments for code you not need.

File: src/components/TimerFocusTask.js (hunk 1 above)
Comment (timodilg) [reply src/components/TimerFocusTask.js]: Done!

File: src/pages/Timer.js (hunk 8 above)
Comment (timodilg) [reply src/pages/Timer.js]: Thanks for the hint. I will change the class component to a functional component in another branch.

**Pull request merged**